apimodels/agent_models.go

```diff
@@ -15,7 +15,7 @@ const (
 	ScopeBuild                      = "build"
 	DefaultSetupTimeoutSecs         = 600
 	DefaultTeardownTimeoutSecs      = 21600
-	DefaultContainerWaitTimeoutSecs = 600
+	DefaultContainerWaitTimeoutSecs = 900
```

Comment:
15 minutes seems like a long time. I don't understand why we need to stretch the default longer.

Reply:
Some of the images we use take a _really_ long time to import. We could keep the default the same though and expect people to specify their own wait time, it is already pretty long.